pkgs/servers/home-assistant/default.nix

```diff
@@ -32,13 +32,6 @@ let
     (mkOverride "pyowm" "2.10.0"
       "1xvcv3sbcn9na8cwz21nnjlixysfk5lymnf65d1nqkbgacc1mm4g")
 
-    # required by aioesphomeapi
-    (self: super: {
-      protobuf = super.protobuf.override {
-        protobuf = protobuf3_6;
```

Comment:
Home Assistant still pins protobuf to 3.6.1: https://github.com/home-assistant/core/blob/0.111.4/requirements_all.txt#L1115
I don't know if it works with a newer version

Reply:
I think that pin is unnecessary. It seems it is pinned as a dependency of tensorflow. Currently, tensorflow is using protobuf 3.8 in nixpkgs. For me, the package builds even with protobuf 3.12, but I have not tested it. Maybe you can try it as a maintainer?

Reply:
Will move the change to other PR